Stripe/STPPaymentMethodParams.m

```diff
@@ -0,0 +1,47 @@
+//
+//  STPPaymentMethodParams.m
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/6/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPaymentMethodParams.h"
+#import "STPPaymentMethod.h"
+
+@implementation STPPaymentMethodParams
+
+@synthesize additionalAPIParameters = _additionalAPIParameters;
+
++ (STPPaymentMethodParams *)paramsWithCard:(STPPaymentMethodCardParams *)card billingDetails:(STPPaymentMethodBillingDetails *)billingDetails metadata:(NSDictionary<NSString *,NSString *> *)metadata {
+    STPPaymentMethodParams *params = [STPPaymentMethodParams new];
+    params.type = STPPaymentMethodParamsTypeCard;
+    params.card = card;
+    params.billingDetails = billingDetails;
+    params.metadata = metadata;
+    return params;
+}
+
+- (NSString *)apiStringType {
```

Comment:
Nice to have would be to follow the pattern that `STPSourceParams` uses re: `type` & `rawTypeString`.